src/components/ui/TripCard/TripCardAccordion.tsx

```diff
@@ -9,6 +7,7 @@ import seatIcon from '../../../assets/SeatIcon.svg';
 import sleeperIcon from '../../../assets/SleeperIcon.svg';
 import Stack from '@mui/material/Stack';
 import { TripAccordionStyled } from './TripCard.styled';
+import { TripCardDetails } from './AccordianDetails/TripCardDetails';
 
 interface ITripCardAccordion {
```

Comment:
Please correct the spelling "Accordian" to "Accordion" in AccordianDetails folder, assets and if present elsewhere.

Reply:
Resolved in 717b41915f078c22341fc331491f0017361434f0